code/datums/trading/ai.dm

```diff
@@ -27,24 +27,66 @@ They sell generic supplies and ask for generic supplies.
 	)
 	possible_wanted_items = list(
 		/obj/item/device/                        = TRADER_SUBTYPES_ONLY,
+		/obj/item/device/kit					 = TRADER_BLACKLIST_ALL, // They're impossible to make or get outside of mining, just annoys traders.
 		/obj/item/device/assembly                = TRADER_BLACKLIST_ALL,
 		/obj/item/device/assembly_holder         = TRADER_BLACKLIST_ALL,
-		/obj/item/device/encryptionkey/syndicate = TRADER_BLACKLIST,
+		/obj/item/device/encryptionkey			 = TRADER_BLACKLIST_ALL, //Why does ai want NT encryption keys?
 		/obj/item/device/radio                   = TRADER_BLACKLIST_ALL,
 		/obj/item/device/pda                     = TRADER_BLACKLIST_SUB,
-		/obj/item/device/uplink                  = TRADER_BLACKLIST
+		/obj/item/device/chameleon				 = TRADER_BLACKLIST, // Why should it want a chameleon projector
+		/obj/item/device/dociler				 = TRADER_BLACKLIST, //Item unobtaineable
```

Comment:
Can be acquired from mining

Reply:
Dociler is not available in the mining vendor. I just went to check to be sure. Unless you mean random abandoned crates which - maybe. I don't know. However, then it's basically unobtaineable.

Reply:
The only thing related to pets in the mining vendor is a lazarus injector.

Reply:
Hm, I thought we had the dociler in there aswell.